packages/server/src/user/user.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Mutation, Args, Resolver, Query, ID } from '@nestjs/graphql';
+import { UserService } from './user.service';
+import { UserModel } from './model/user.model';
+import { User } from '@prisma/client';
+
+@Resolver()
+export class UserResolver {
+  constructor(private readonly userService: UserService) {}
+
+  @Query(() => UserModel)
```

Comment:
```suggestion
  @Query(() => [UserModel])
```
We need to specify to graphql that this is an array.